bin/update_dependencies.py

```diff
@@ -20,58 +20,73 @@
-        subprocess.run([
-            'docker', 'run', '--rm',
-            '-e', 'CUSTOM_COMPILE_COMMAND',
-            '-v', f'{os.getcwd()}:/volume',
-            '--workdir', '/volume', image_runner,
-            'bash', '-c',
-            f'{python_path}pip install pip-tools &&'
-            f'{python_path}pip-compile --allow-unsafe --upgrade '
-            'cibuildwheel/resources/constraints.in '
-            f'--output-file cibuildwheel/resources/constraints-python{python_version}.txt'
-        ], check=True)
+        subprocess.run(
+            [
+                'docker',
+                'run',
+                '--rm',
+                '-e',
+                'CUSTOM_COMPILE_COMMAND',
+                '-v',
+                f'{os.getcwd()}:/volume',
+                '--workdir',
+                '/volume',
+                image_runner,
+                'bash',
+                '-c',
+                f'{python_path}pip install pip-tools &&'
+                f'{python_path}pip-compile --allow-unsafe --upgrade '
+                'cibuildwheel/resources/constraints.in '
+                f'--output-file cibuildwheel/resources/constraints-python{python_version}.txt',
+            ],
+            check=True,
+        )
 
 # default constraints.txt
-shutil.copyfile(f'cibuildwheel/resources/constraints-python{PYTHON_VERSIONS[-1]}.txt', 'cibuildwheel/resources/constraints.txt')
-
-Image = namedtuple('Image', [
-    'manylinux_version',
-    'platform',
-    'image_name',
-    'tag',
-])
+shutil.copyfile(
+    f'cibuildwheel/resources/constraints-python{PYTHON_VERSIONS[-1]}.txt', 'cibuildwheel/resources/constraints.txt'
+)
+
+Image = namedtuple(
+    'Image',
+    [
+        'manylinux_version',
+        'platform',
+        'image_name',
+        'tag',
+    ],
+)
 
 images = [
     Image('manylinux1', 'x86_64', 'quay.io/pypa/manylinux1_x86_64', None),
     Image('manylinux1', 'i686', 'quay.io/pypa/manylinux1_i686', None),
-
     # Images for manylinux2010 are pinned to the latest tag supporting cp27
     Image('manylinux2010', 'x86_64', 'quay.io/pypa/manylinux2010_x86_64', '2021-02-06-3d322a5'),
     Image('manylinux2010', 'i686', 'quay.io/pypa/manylinux2010_i686', '2021-02-06-3d322a5'),
-
     Image('manylinux2010', 'pypy_x86_64', 'pypywheels/manylinux2010-pypy_x86_64', None),
-
     Image('manylinux2014', 'x86_64', 'quay.io/pypa/manylinux2014_x86_64', None),
     Image('manylinux2014', 'i686', 'quay.io/pypa/manylinux2014_i686', None),
     Image('manylinux2014', 'aarch64', 'quay.io/pypa/manylinux2014_aarch64', None),
     Image('manylinux2014', 'ppc64le', 'quay.io/pypa/manylinux2014_ppc64le', None),
     Image('manylinux2014', 's390x', 'quay.io/pypa/manylinux2014_s390x', None),
-
     Image('manylinux_2_24', 'x86_64', 'quay.io/pypa/manylinux_2_24_x86_64', None),
     Image('manylinux_2_24', 'i686', 'quay.io/pypa/manylinux_2_24_i686', None),
     Image('manylinux_2_24', 'aarch64', 'quay.io/pypa/manylinux_2_24_aarch64', None),
```

Comment:
This is still readable without the extra lines, but if we wanted to keep it, we can add fmt on/off comments.